src/main/scala/Configs.scala

```diff
@@ -50,6 +50,30 @@ class WithNBlockDeviceTrackers(n: Int) extends Config((site, here, up) => {
   }
 })
 
-class WithTSI extends Config((site, here, up) => {
-  case SerialKey => true
+// Default size should be tiny
+class WithDefaultSerialTL extends Config((site, here, up) => {
+  case SerialTLKey => Some(SerialTLParams(
+    memParams = MemoryPortParams(MasterPortParams(
+      base = BigInt("60000000", 16),
```

Comment:
I 'm not sure I like this location.

Reply:
This is to avoid conflicting with the default backing AXI4 memory.
When using this memory channel as the main backing memory, the address should be set to 0x80000000

Reply:
Mainly my question is why even have this default? If the user is expected to configure it differently in the common use case it seems likely there will be bugs where someone mixes in the wrong config class.
If we had a `p(SerialMem)` then the user would get a run time error if they haven't configured it properly.

Reply:
Would you prefer the default case to be no external LBWIF memory? With a config fragment `WithSerialBaseMem` that sets `SerialMem` to `ExtMem` and clears `ExtMem`?